<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTRL_Levels</class>
 <widget class="QWidget" name="CTRL_Levels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>576</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>576</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contrast and W&amp;L</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/gimp-pro/images/gimp-histogram.png</normaloff>:/resources/gimp-pro/images/gimp-histogram.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Linear Contrast Adjustment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_level"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ma&amp;ximum</string>
        </property>
        <property name="buddy">
         <cstring>le_max</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wi&amp;ndow</string>
        </property>
        <property name="buddy">
         <cstring>le_win</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_min"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="le_win"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_max"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Minimum</string>
        </property>
        <property name="buddy">
         <cstring>le_min</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Le&amp;vel</string>
        </property>
        <property name="buddy">
         <cstring>le_level</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="btn_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QToolButton" name="btn_auto">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>570</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>570</width>
       <height>450</height>
      </size>
     </property>
     <property name="title">
      <string>Curve based Contrast Adjustment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
